backend/src/models/mural.model.ts

```diff
@@ -98,4 +108,7 @@ export interface MuralInterface {
   description?: string;
   partners?: string[];
   assistants?: string[];
+  longitude?: number;
+  latitude?: number;
```

Comment:
longitude/latitude or coordinates?

Reply:
the best practice i saw for creating something with a postgis coordinate field was to pass longitude and latitude in the api call, and then to create a postgis coordinate point using those which is then passed to the Mural.create() call, check the newest mural test case for an example